news/_posts/core-weekly/2019-02-11-coreweekly-week-06-2019.md

```diff
@@ -18,7 +18,7 @@ This edition of the Core Weekly report highlights changes in PrestaShop's core c
 
 Dear Developers,
 
-Last week, Pablo Borowicz, PrestaShop's current lead developer, started [a series on the blog about the past, present and future of the PrestaShop open source project](http://build.prestashop.com/news/prestashop-in-2019-and-beyond-introduction/). The next chapter is currently under review and should be published this week.
+Last week, Pablo Borowicz, PrestaShop's lead developer, started [a series on the blog about the past, present and future of the PrestaShop open source project](http://build.prestashop.com/news/prestashop-in-2019-and-beyond-introduction/). The next chapter is currently under review and should be published this week.
```

Comment:
"past, present and future of the PrestaShop open source project"

Still not the right title (unless I'm missing something)

Reply:
"about the architecture of the PrestaShop open source project" ?